src/boss/mar/AFC4.c

```diff
@@ -11,14 +11,640 @@ extern Dialogue g_Dialogue;
 
 #include "../../st/cutscene_unk4.h"
 
-INCLUDE_ASM("boss/mar/nonmatchings/AFC4", func_us_8018B2F0);
+extern char* D_us_80180680[];
+
+void func_us_8018B2F0(u16 actorIndex, Entity* self) {
```

Comment:
Could we factor this out? Doing it in this PR is probably going to make it too big though